plugins/modules/vcenter_root_password_expiration.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Ansible Project
+# Copyright: (c) 2023, Valentin Yonev <valentin.ionev@live.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+DOCUMENTATION = r'''
+---
+module: vcenter_root_password_expiration
```

Comment:
```suggestion
module: vcenter_root_password_expiration
version_added: '3.10.0'
```

Sorry, I didn't have the time to have a closer look at your code yet. But we should at least document when this module has been added.

I don't think we need a [changelog fragment](https://docs.ansible.com/ansible/latest/community/collection_development_process.html#creating-changelog-fragments) here. New modules should show up without one automatically in the changelog. But `version_added` should show up in the module documentation and tell people since when it's available.